src/com/google/enterprise/adaptor/GsaFeedFileSender.java

```diff
@@ -257,12 +259,12 @@ public void sendFull(String datasource, String xmlString,
    * Groupsource name is limited to [a-zA-Z_][a-zA-Z0-9_-]*.
    */
   void sendGroups(String groupsource, String xmlString,
-      boolean useCompression) throws IOException {
+      boolean useCompression, boolean incremental) throws IOException {
```

Comment:
From this context, I definitely don't like the order. incremental before useCompression or xmlString would feel better to me. This order pairs (A, B, D) with (A, B, C, D), rather than (A, B, D, C).